Sources/Testing/ExitTests/ExitTest.Condition.swift

```diff
@@ -0,0 +1,148 @@
+  ///
+  /// | Platform | Header |
+  /// |-|-|
+  /// | macOS | [`<stdlib.h>`](https://developer.apple.com/library/archive/documentation/System/Conceptual/ManPages_iPhoneOS/man3/_Exit.3.html), [`<sysexits.h>`](https://developer.apple.com/library/archive/documentation/System/Conceptual/ManPages_iPhoneOS/man3/sysexits.3.html) |
+  /// | Linux | [`<stdlib.h>`](https://sourceware.org/glibc/manual/latest/html_node/Exit-Status.html), `<sysexits.h>` |
+  /// | FreeBSD | [`<stdlib.h>`](https://man.freebsd.org/cgi/man.cgi?exit(3)), [`<sysexits.h>`](https://man.freebsd.org/cgi/man.cgi?sysexits(3)) |
+  /// | OpenBSD | [`<stdlib.h>`](https://man.openbsd.org/exit.3), [`<sysexits.h>`](https://man.openbsd.org/sysexits.3) |
+  /// | Windows | [`<stdlib.h>`](https://learn.microsoft.com/en-us/cpp/c-runtime-library/exit-success-exit-failure) |
+  ///
+  /// On macOS, FreeBSD, OpenBSD, and Windows, the full exit code reported by
+  /// the process is yielded to the parent process. Linux and other POSIX-like
+  /// systems may only reliably report the low unsigned 8 bits (0&ndash;255) of
+  /// the exit code.
+  public static func exitCode(_ exitCode: CInt) -> Self {
+#if !SWT_NO_EXIT_TESTS
+    Self(.exitCode(exitCode))
+#else
+    fatalError("Unsupported")
+#endif
+  }
+
+  /// The process terminated with the given signal.
+  ///
+  /// - Parameters:
+  ///   - signal: The signal that terminated the process.
+  ///
+  /// The C programming language defines a number of [standard signals](https://en.cppreference.com/w/c/program/SIG_types).
+  /// Platforms may additionally define their own non-standard signal codes:
+  ///
+  /// | Platform | Header |
+  /// |-|-|
+  /// | macOS | [`<signal.h>`](https://developer.apple.com/library/archive/documentation/System/Conceptual/ManPages_iPhoneOS/man3/signal.3.html) |
+  /// | Linux | [`<signal.h>`](https://sourceware.org/glibc/manual/latest/html_node/Standard-Signals.html) |
+  /// | FreeBSD | [`<signal.h>`](https://man.freebsd.org/cgi/man.cgi?signal(3)) |
+  /// | OpenBSD | [`<signal.h>`](https://man.openbsd.org/signal.3) |
+  /// | Windows | [`<signal.h>`](https://learn.microsoft.com/en-us/cpp/c-runtime-library/signal-constants) |
+  public static func signal(_ signal: CInt) -> Self {
+#if !SWT_NO_EXIT_TESTS
+    Self(.signal(signal))
+#else
+    fatalError("Unsupported")
+#endif
+  }
+}
+
+// MARK: - Comparison
+
+#if SWT_NO_EXIT_TESTS
+@available(*, unavailable, message: "Exit tests are not available on this platform.")
+#endif
+extension ExitTest.Condition {
+  /// Check whether or not an exit test condition matches a given exit status.
+  ///
+  /// - Parameters:
+  ///   - other: Another value to compare.
+  ///
+  /// - Returns: Whether or not `self` and `other` are equal.
+  ///
+  /// Two exit test requirements can be compared; if either instance is equal to
+  /// ``failure``, it will compare equal to any instance except ``success``.
+  func isApproximatelyEqual(to statusAtExit: StatusAtExit) -> Bool {
```

Comment:
Since this type is called "Condition", maybe `isSatisfied(by:)` would be more fitting? I find it confusing to be considering this a form of _equality_; I think it's really more like "this status _satisfies_ this condition", or "_meets_", or something like that.

Reply:
This is an internal function and when you look at the use case, I don't think it's terribly confusion. Do you feel strongly?

Reply:
I certainly don't mind as much after realizing it's not API, but I do think a different phrase would be more clear. To me, if you have a condition you evaluate it against something. If you say one condition "is approximately equal" to something else, I would assume that something else would itself be a condition. But the parameter is a status, i.e. a subject with which you evaluate the condition.